app/models/cart.rb

```diff
@@ -0,0 +1,4 @@
+class Cart < ApplicationRecord
+  has_many :cart_products
+  has_many :orders
```

Comment:
ủa chị nhớ cart ko lưu vào DB mà dùng session mà chú

Reply:
e có lưu cart_id trong session, e dùng bảng orders để kết nối giữa users và carts . vì lúc trước nếu người dùng chưa đăng nhập mà chọn sản phẩm thì em không tạo được order vì order cần user_id, nên e tạo bảng cart để hiển thị danh sách đã chọn, khi nào khách hàng thanh toán thì mới nhập thông tin cart vào order ạ